### ISSUE TITLE
[NMT] Swift Package Manager not working

### ISSUE THREAD

**Issue by matance** [SourcePointUSA/ios-cmp-app#265]
**Describe the bug**
When using Swift Package Manager to integrate the SDK, the Consent Layer does not show and following messages are available:

**Resolve Package Graph:**
```
found 3 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
[…]/SourcePackages/checkouts/ios-cmp-app/ConsentViewController/Assets/javascript/jest.config.json
[…]/SourcePackages/checkouts/ios-cmp-app/ConsentViewController/Assets/javascript/GDPRJSReceiver.js
[…]/SourcePackages/checkouts/ios-cmp-app/ConsentViewController/Assets/javascript/GDPRJSReceiver.spec.js
```

When the app starts, following error is shown:
`onError: Unable to load the JSReceiver.js resource.`

**About your campaign (please complete the following information):**
- Account Id: 75
- Property Id 5593
- Property Name BILD-NEWS-IOS
- PrivacyManager Id: 134100

** About your environment **
 - SDK version 5.3.2
 - Device: iPhone 11 Simulator
 - OS: iOS 14.3

** To Reproduce**
1. Integrate the SDK using Swift Package Manager
2. Start the app in the simulator
3. Check the build time warnings
4. Check the `onError(error: GDPRConsentViewControllerError?)` method

**Expected behavior**
No buildtime warnings. Consent layer should appear.

**Comment by andresilveirah** [comment]
Hi @matance thank you for reporting this issue. It's indeed a regression. We are working on a fix and it'll be release as soon as possible.

**Comment by andresilveirah** [comment]
@matance we have just released version `5.3.3` with a fix for this issue. Can you give it a try and let me know how it went?

**Comment by matance** [comment]
@andresilveirah : I did a quick check and it seems to be working now! Thanks for the fast fixing of the issue!
